pyproject.toml

```diff
@@ -22,7 +22,6 @@ django-bootstrap-icons = "^0.8.2"
 [tool.poetry.group.dev.dependencies]
 black = "^23.1.0"
 flake8 = "^5.0.4"
-coverage = "^6.5.0"
```

Comment:
I know coveralls would add `coverage` but I think it is still good to have it as a dependency. Could you please revert this change and update the `poetry.lock` file accordingly by running
```
$ poetry lock
```